internal/cri/store/container/status.go

```diff
@@ -271,7 +306,12 @@ func (s *statusStorage) UpdateSync(u UpdateFunc) error {
 		return fmt.Errorf("failed to encode status: %w", err)
 	}
 	if err := continuity.AtomicWriteFile(s.path, data, 0600); err != nil {
-		return fmt.Errorf("failed to checkpoint status to %q: %w", s.path, err)
+		if strings.Contains(err.Error(), "no space") {
```

Comment:
`errors.Is(err, unix.ENOSPC)` may work?

Reply:
Different platforms need to be used here, and platform file may be required for error handling.

If the plan is confirmed to be feasible,maybe should continue to modify it here.

Reply:
use `syscall.ENOSPC` which is depending on the underlying system